### PR TITLE
kernel: bump 6.6 to 6.6.32

### PR DESCRIPTION
**Pull request by graysky2**
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.32

Only kernel checksum was updated/no patches needed a rebase.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne, flogic/glinet_gl-mt6000, flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3
Run-tested: x86/64/AMD Cezanne, flogic/glinet_gl-mt6000, flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3

### CODE REVIEW

**Comment by Ansuel** [comment]
The pending patch requires some extra check

**Comment by graysky2** [comment]
Thanks @Ansuel - please advise when you have time to review.

**Comment by graysky2** [comment]
@rmilecki - Rafał can you please comment on the removal of your patch in context with upstream changes?  Is removing it the correct course of action or should I merge it taking the change in your patch?

File: target/linux/generic/pending-6.6/778-net-l2tp-drop-flow-hash-on-forward.patch [hunk 1]
```diff
@@ -19,9 +19,9 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
...
 +++ b/net/l2tp/l2tp_eth.c
-@@ -136,6 +136,9 @@ static void l2tp_eth_dev_recv(struct l2t
- 	/* checksums verified by L2TP */
- 	skb->ip_summed = CHECKSUM_NONE;
+@@ -139,6 +139,9 @@ static void l2tp_eth_dev_recv(struct l2t
+ 	/* drop outer flow-hash */
+ 	skb_clear_hash(skb);
  
 +	/* drop outer flow-hash */
 +	skb_clear_hash(skb);
```
Comment (DragonBluep) [inline target/linux/generic/pending-6.6/778-net-l2tp-drop-flow-hash-on-forward.patch:27]: It seems that this patch has been upstreamed. Please check it again.

Comment (graysky2) [reply target/linux/generic/pending-6.6/778-net-l2tp-drop-flow-hash-on-forward.patch:27]: Interesting that the script did not find this.  I ran it again as a test.  Did you identify this manually or did your script identify this?

Comment (DragonBluep) [reply target/linux/generic/pending-6.6/778-net-l2tp-drop-flow-hash-on-forward.patch]: No script, just some little experience haha~

File: target/linux/generic/pending-6.6/684-net-bridge-fix-corrupted-ethernet-header-on-multicas.patch [hunk 2]
```diff
@@ -1,42 +0,0 @@
...
-@@ -266,12 +267,16 @@ static void maybe_deliver_addr(struct ne
- 	if (skb->dev == p->dev && ether_addr_equal(src, addr))
- 		return;
- 
--	skb = pskb_copy(skb, GFP_ATOMIC);
--	if (!skb) {
-+	__skb_push(skb, ETH_HLEN);
-+	nskb = pskb_copy(skb, GFP_ATOMIC);
-+	__skb_pull(skb, ETH_HLEN);
-+	if (!nskb) {
```
Comment (hauke) [inline target/linux/generic/pending-6.6/684-net-bridge-fix-corrupted-ethernet-header-on-multicas.patch:33]: This is upstream in this commit:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.31&id=1ce60741d7e17dcf707e3bdebed4442ae30bdd02
The reference in your commit message is wrong.

File: target/linux/generic/pending-6.6/440-mtd-don-t-look-for-OTP-legacy-NVMEM-cells-if-proper-.patch [hunk 3]
```diff
@@ -1,44 +0,0 @@
...
-Reported-by: Christian Marangi <ansuelsmth@gmail.com>
-Fixes: 4b361cfa8624 ("mtd: core: add OTP nvmem provider support")
-Fixes: e87161321a40 ("mtd: rawnand: macronix: OTP access for MX30LFxG18AC")
-Signed-off-by: Rafał Miłecki <rafal@milecki.pl>
----
- drivers/mtd/mtdcore.c | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
---- a/drivers/mtd/mtdcore.c
-+++ b/drivers/mtd/mtdcore.c
```
Comment (hauke) [inline target/linux/generic/pending-6.6/440-mtd-don-t-look-for-OTP-legacy-NVMEM-cells-if-proper-.patch:35]: The commit description of this upstream commit sounds very similar:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.31&id=cce311f8bbbf9d24eacc39af3228cd104e079396

I assume that this is the upstream version of this patch in OpenWrt. 
@rmilecki please clarify.

File: target/linux/generic/pending-6.6/684-net-bridge-fix-corrupted-ethernet-header-on-multicas.patch (hunk 2 above)
Comment (graysky2) [reply target/linux/generic/pending-6.6/684-net-bridge-fix-corrupted-ethernet-header-on-multicas.patch:33]: Fixed

File: target/linux/generic/pending-6.6/440-mtd-don-t-look-for-OTP-legacy-NVMEM-cells-if-proper-.patch (hunk 3 above)
Comment (graysky2) [reply target/linux/generic/pending-6.6/440-mtd-don-t-look-for-OTP-legacy-NVMEM-cells-if-proper-.patch:35]: Thanks all.  As you see for the commit message, I removed this patch in the initial commit.  Let me know if anything else is keeping this PR from getting merged.

Comment (hauke) [reply target/linux/generic/pending-6.6/440-mtd-don-t-look-for-OTP-legacy-NVMEM-cells-if-proper-.patch:35]: Please update the reference to this commit in the commit message too.

Comment (graysky2) [reply target/linux/generic/pending-6.6/440-mtd-don-t-look-for-OTP-legacy-NVMEM-cells-if-proper-.patch:35]: Please see my latest push and let me know if it needs to be simplified to just the commit (I split it into two references)

Comment (hauke) [reply target/linux/generic/pending-6.6/440-mtd-don-t-look-for-OTP-legacy-NVMEM-cells-if-proper-.patch:35]: looks fine

**Pull request merged**